### PR TITLE
add client information for each session

### LINKED ISSUE #453: List all sessions
For each session, we'd like to have the following informations:  version, ip, hostname, tags, locks, heartbeat

### PR DESCRIPTION
**Pull request by Sandy4999**
close #453

### CODE REVIEW

File: pkg/meta/interface.go [hunk 1]
```diff
@@ -261,3 +296,16 @@ func NewClient(uri string, conf *Config) Meta {
...
+
+func newSessionInfo() (*SessionInfo, error) {
+	host, err := os.Hostname()
+	if err != nil {
+		return nil, err
+	}
+	return &SessionInfo{
+		Version:   version.Version(),
+		Hostname:  host,
+		Tags:      make([]string, 0),
```
Comment (davies) [inline pkg/meta/interface.go:308]: Just ignore it as empty

File: pkg/meta/info.go [hunk 2]
```diff
@@ -21,17 +21,17 @@ import (
 	"strings"
 )
 
-type version struct {
+type redisVersion struct {
```
Comment (davies) [inline pkg/meta/info.go:24]: pleasse merge wit main branch

File: cmd/status.go [hunk 3]
```diff
@@ -49,5 +65,11 @@ func statusFlags() *cli.Command {
 		Usage:     "show status of JuiceFS",
 		ArgsUsage: "REDIS-URL",
 		Action:    status,
+		Flags: []cli.Flag{
+			&cli.BoolFlag{
+				Name:  "detail",
```
Comment (davies) [inline cmd/status.go:70]: display details for single session

Comment (Sandy4999) [reply cmd/status.go:70]: This flag will be removed

**Pull request merged**